packages/web_benchmarks/lib/src/runner.dart

```diff
@@ -156,7 +155,7 @@ class BenchmarkServer {
           // Trace data is null when the benchmark is not frame-based, such as RawRecorder.
           if (latestPerformanceTrace != null) {
             final BlinkTraceSummary traceSummary =
-                BlinkTraceSummary.fromJson(latestPerformanceTrace);
+                BlinkTraceSummary.fromJson(latestPerformanceTrace!)!;
```

Comment:
Throughout this function, look for opportunities to use locals so that the checks don't require force unwrapping.

Reply:
` BlinkTraceSummary.fromJson` has not many things that can be replaced with locals, it returns null only in one case.